iron-autogrow-textarea.html

```diff
@@ -80,32 +70,12 @@
     ::content textarea:invalid {
       box-shadow: none;
     }
-
-    textarea::-webkit-input-placeholder {
-      @apply(--iron-autogrow-textarea-placeholder);
-    }
-
-    textarea:-moz-placeholder {
-      @apply(--iron-autogrow-textarea-placeholder);
-    }
-
-    textarea::-moz-placeholder {
-      @apply(--iron-autogrow-textarea-placeholder);
-    }
-
-    textarea:-ms-input-placeholder {
-      @apply(--iron-autogrow-textarea-placeholder);
-    }
   </style>
   <template>
-    <!-- the mirror sizes the input/textarea so it grows with typing -->
-    <!-- use &#160; instead &nbsp; of to allow this element to be used in XHTML -->
-    <div id="mirror" class="mirror-text" aria-hidden="true">&#160;</div>
-
     <!-- size the input/textarea with a div, because the textarea has intrinsic size in ff -->
-    <div class="textarea-container fit">
       <textarea id="textarea"
         name$="[[name]]"
+        value="{{value::input}}"
```

Comment:
This isn't entirely correct -- updating the value imperatively won't fire an `input` event, which means this won't be updated correctly in those cases